src/schema/facet.rs

```diff
@@ -40,6 +40,7 @@ pub enum FacetParseError {
 /// belonging to a facet also belongs to the ancestor of
 /// its facet. In the example above, `/electronics/tv_and_video/`
 /// and `/electronics`.
+#[repr(transparent)]
```

Comment:
Why do we need this?

Reply:
This got left over from when I was trying to work around the facet being an owned value making it somewhat awkward to use on borrowed data without allocating each time.

I ended up giving up because making it use borrowed data made things worse, so this can be removed.